system/lib/libc/emscripten_mmap.c

```diff
@@ -126,13 +128,14 @@ intptr_t __syscall_mmap2(intptr_t addr, size_t len, int prot, int flags, int fd,
   // but it is widely used way to allocate memory pages on Linux, BSD and Mac.
   // In this case fd argument is ignored.
   if (flags & MAP_ANONYMOUS) {
+    size_t alloc_len = ALIGN_TO(len, 16);
```

Comment:
Can this be `alignof(struct map)` instead or a magic constant?

Reply:
It only needs to be aligned to 4/8 tbh, so the fields of the struct are aligned.

Reply:
Yes.